server/api/router/main.go

```diff
@@ -80,25 +84,36 @@ func RouteCredentials(parent *echo.Group, persister persistence.Persister) {
 }
 
 func RouteRegistration(parent *echo.Group, persister persistence.Persister, authenticatorMetadata mapper.AuthenticatorMetadata) {
-	registrationHandler := handler.NewRegistrationHandler(persister, authenticatorMetadata)
+	registrationHandler := handler.NewRegistrationHandler(persister, authenticatorMetadata, false)
 
-	group := parent.Group("/registration", passkeyMiddleware.WebauthnMiddleware())
+	group := parent.Group("/registration", passkeyMiddleware.WebauthnMiddleware(persister))
```

Comment:
```suggestion
	group := parent.Group("/registration")
```
Because the WebAuthnMiddleware is already added to the parent group.